CMakeLists.txt

```diff
@@ -57,11 +57,70 @@ endif()
 
 list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_LIST_DIR}/cmake)
 
+# To configure mbedtls
```

Comment:
seems like these are the same, and really i guess the TODO should be in the header